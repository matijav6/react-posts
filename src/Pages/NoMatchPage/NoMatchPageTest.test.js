import renderer from 'react-test-renderer';
import {NoMatchPage} from './';

test('No match page renders correctly', () => {
    const tree = renderer
        .create(<NoMatchPage/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
