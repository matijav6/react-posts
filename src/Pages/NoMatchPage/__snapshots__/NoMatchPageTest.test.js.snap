// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No match page renders correctly 1`] = `
Array [
  <h1>
    Oops, seems like this page does not exist
  </h1>,
  <button
    className="btn btn-primary"
    onClick={[Function]}
  >
    Back to posts
  </button>,
]
`;
